modules/monitoring-setting-log-levels-for-monitoring-components.adoc

```diff
@@ -19,17 +29,24 @@ The default log level is `info`.
 
 .Prerequisites
 
+ifndef::openshift-dedicated,openshift-rosa[]
```

Comment:
I found this content in Platform, but not ROSA or Dedicated. 

Also Prometheus is listed twice in several places in Platform. In the other previews, the double Prometheus appears once, just after the top level heading.